### DISCUSSION TITLE
SiLabs Gecko SDK 5.0.0.0 + Silicon Labs Simplicity SDK Suite 2024.6.0 has been announced and to be released soon and support for EZR32/EFM32/EFR32 Series 0 and Series 1 devices been removed

### DISCUSSION THREAD

**Discussion by Hedda** [zwave-js/zwave-js#6947] [General]
Just a heads-up, sounds as if Silicon Labs is phasing out Z-Wave 700 and older by only including Z-Wave 800 in the new SDK versions.

FYI, Silicon Labs Gecko SDK 5.0.0.0 + SiLabs Simplicity SDK Suite 2024.6.0 been announced (but not yet released but will very soon?):

https://www.silabs.com/developers/software-development-kit-release-notes

https://www.silabs.com/developers/simplicity-software-development-kit

Gecko SDK 5.0 seems to be scheduled to be released very soon as Q2/2024 (June 2024) is specifically mentioned as the release date:

https://www.silabs.com/documents/public/release-notes/gecko-platform-release-notes-5.0.0.0.pdf

https://www.silabs.com/developers/simplicity-software-development-kit

https://github.com/SiliconLabs/gecko_sdk

https://github.com/SiliconLabs/simplicity_sdk

Perhaps most noteworthy is that SiLabs Gecko SDK 5.0 / 5.x has removed support for EZR32/EFM32/EFR32 Series 0 and Series 1 devices.

Gecko Platform 5.0 will remove support for EFR32 Series 0/1 (e.i. EFR32MGx and EFR32MG1x) so need to use separate SDKs for Series 0 & 1 and Series 2 & 3 Starting in Q2 2024 (June 2024) as the new Gecko Platform and SDK are designed specifically for our EFR32 Series 2 and upcoming Series 3 devices only.

It is however still unclear to me which specific Z-Wave SoC chips will not be supported in Gecko Platform 5.x and newer versions.

If I understand it then it sounds like only Z-Wave 800 Series SoCs will be supported in SiLabs Gecko SDK 5.0 / 5.x platform versions?

So only Z-Wave 800 will be supported in Z-Wave SDK version 7.22.0.0+ while Z-Wave 700 and older will only continue in 7.21.x.x? 

https://community.silabs.com/s/article/series-0-and-series-1-efm32-ezr32-efr32-device?language=en_US

Also newly released are Z-Wave and Z-Wave Long Range 800 version 7.22.0.0 which changelog says something along those lines.

If so then it means Z-Wave 700 and older SoCs will continue with 7.2x.x.x as long-term support for bug-fixes but not get new features?

https://www.silabs.com/documents/public/release-notes/SRN14930-7.22.0.0.pdf

#### New in Z-Wave and Z-Wave Long Range 800 SDK version 7.22.0.0

- 7.22.0 GA is self-certified according to the approved 2023B Specification test suite
- **7.22.0 and future updates support the 800 Series platform**
- **700 Series platform will continue to be supported through the upcoming 7.21.x releases**
- Introducing ZWLR Europe support in experimental quality
- Z-Wave NCP controller stability improvements
- Introducing User Credential Command Class beta
- Introducing CLI support in sample apps
- Removed support for Series 0/1

![image](https://github.com/zigpy/zigpy/assets/6320001/24124a98-49d1-448f-9dfc-19ae5a6fb58a)

"_Meanwhile, our Gecko Software Development Kit (GSDK) will continue to be available for users of our Series 0 and Series 1 devices. The Gecko SDK will continue to support all [Series 0 and 1 devices](https://www.silabs.com/developers/software-development-kit-release-notes/series-directory) with no change to the long-term support, maintenance, quality, and responsiveness provided under our software policy.The new SDK will branch from Gecko SDK and begin to offer new features that help developers take advantage of the advanced capabilities of our Series 2 and 3 products._"

### New in Gecko Platform Version 5.0.0.0 (Simplicity SDK Version 2024.6.0)

* Migration to Simplicity SDK Suite
* Removed support for Series 0/1
 * Peripherals
  * New API names introduced for low-level drivers (with compatibility layer for old names)
* Services
  * Memory Manager, APIs to manage embedded dynamic memory allocations
  * Clock Manager, for clock tree initialization
  * Event System, for inter-process communication
  * Interrupt manager, APIs to manage embedded interrupts
  * APIs for accessing value of tokens added to Token Manager
* CPC
  * Small improvement and bug fixes
* Security
  * Mbed TLS upgraded (to version 3.5.0)
* RAIL 5.0.0
  * Added support for collision detection feature on the EFR32xG25 parts
  * Added support for additional Coex TX and RX metrics events to RAIL coexistence utility
  * Updated some RAIL APIs to better prepare for future API changes while retaining backwards compile-time compatibility
* Other Components
  * Compilers upgraded (to GCC 12.2.1 and IAR 9.40.1)

PS Also newly released is USB Device Stack 1.3.0.0 version:

https://www.silabs.com/documents/public/release-notes/usb-device-release-notes-1.3.0.0.pdf

#### New in USB Device Stack version 1.3.0.0

- Underlying platform changes only

**Comment by AlCalzone** [comment]
FWIW, Simplicity SDK is already released and available within Simplicity Studio.

Also I don't think Gecko **SDK** will be released as version 5. The links above mention Gecko **Platform** 5.0, which is serviced by Simplicity SDK.

Not sure about their versioning plans. Limiting 700 series to 7.21.x sounds like they'll only receive patch releases, but then I didn't have the impression that the firmware versions follow SemVer anyways.
